### DISCUSSION TITLE
Issue on GIT branch Protection Rule

### DISCUSSION THREAD

**Discussion by Satz-Babu** [pmd/pmd#3634] [Q&A] [answered]
Enabled protection rule to a branch such that only certain users can merge others code, but not their own code.
![image](https://user-images.githubusercontent.com/93532994/141896335-f0ee5c9f-4e44-46ef-abad-fb6594077be9.png)
Issue:
Developer 1 creates a pull request and sends for an approval for merge.
Now Approver modifies the code on the same pull request, **now the approver has the rights to approve/merge the pull request which should not be the case**  
How to avoid this issue?

**Comment by adangel** [comment, marked as answer]
@Satz-Babu thanks for your question. However, this discussion forum is about PMD and not about github. Please direct your question to github support: https://support.github.com/
